pmm-app/package.json

```diff
@@ -20,7 +20,7 @@
     "lint:dev": "eslint --ext .tsx,.ts --fix src/",
     "lint:tests": "eslint --ext .js --fix tests/",
     "jest": "jest",
-    "e2e": "docker run --net=host -v $PWD:/tests -v $PWD/node_modules:/node_modules --env-file env.generated.list codeception/codeceptjs:2.6.1 codeceptjs run-multiple parallel --all --debug --steps --reporter mocha-multi -c pr.codecept.js --grep '(?=.*)^(?!.*@not-ui-pipeline)^(?!.*@not-pr-pipeline)'",
+    "e2e": "./node_modules/.bin/codeceptjs run-multiple parallel --debug --steps --reporter mocha-multi -c pr.codecept.js --grep '(?=.*)^(?!.*@not-ui-pipeline)^(?!.*@not-pr-pipeline)'",
     "e2e:local": "docker run --net=host --name codecept -v $PWD:/tests -v $PWD/node_modules:/node_modules codeception/codeceptjs:2.6.1 codeceptjs run-multiple parallel --all --debug --steps -c pr-local.codecept.js --grep '(?=.*)^(?!.*@not-ui-pipeline)^(?!.*@not-pr-pipeline)'",
```

Comment:
@IvanaBrecherova change this also please, we can not use docker based runs any more